staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -235,8 +233,6 @@ func validateBackendBatchOptions(pluginName string, options AuditBatchOptions) e
 }
 
 var knownGroupVersions = []schema.GroupVersion{
-	auditv1alpha1.SchemeGroupVersion,
```

Comment:
note to self for future me.  This is used a few lines down in a validate method and will prevent the server from starting if the value is anything except v1.